### PR TITLE
Fix link for automated installer

### PR DESCRIPTION
**Pull request by chripede**
The link to the automated installer was for compose-cli, not compose-switch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Compose Switch is a replacement to the Compose V1 `docker-compose` (python) exec
 We provide an script for automated installation:
 
 ```console
-$ curl -fL https://raw.githubusercontent.com/docker/compose-cli/main/scripts/install/install_linux.sh | sh
+$ curl -fL https://raw.githubusercontent.com/docker/compose-switch/master/install_on_linux.sh | sh
```
Comment (ulyssessouza) [inline README.md:11]: LGTM but we could also add the missing `sudo` as you've stated in https://github.com/docker/compose-switch/issues/6

```suggestion
$ curl -fL https://raw.githubusercontent.com/docker/compose-switch/master/install_on_linux.sh | sudo bash -
```

**Review by ulyssessouza — CHANGES_REQUESTED** [review]

**Comment by francislavoie** [comment]
@ndeloof Can we get eyes on this? This mistake caused me to waste quite a lot of time :disappointed:

**Comment by francislavoie** [comment]
@ndeloof has a decision been made about this? I see compose v2.1.0 was released, but the instructions on https://docs.docker.com/compose/cli-command/ are still outdated, and this repo hasn't been touched in 2 months.

It's still not clear how to actually correctly install docker-compose on linux machines with proper compatibility with old scripts retained.

**Comment by ndeloof** [comment]
@francislavoie I left Docker, so I can't tell about decision on linux packaging
cc @ulyssessouza @thaJeztah

**Pull request closed without merging**